next.config.js

```diff
@@ -66,24 +66,14 @@ const nextConfig = {
         }
 
         return (context, callback) => {
-          if (
-            context.request.indexOf('@elastic/eui') > -1 ||
-            context.request.indexOf('react-ace') > -1
-          ) {
+          if (context.request.indexOf('@elastic/eui') > -1) {
```

Comment:
Can we check whether we still need any of this `config.externals` customisation now?

Reply:
For sure. I'm not even sure what this was trying to do in the first place 😅

Reply:
I was something to do with SSR and react-ace...beyond that I don't really remember 🤷 😀

Reply:
The `context.request.indexOf('@elastic/eui') > -1` check does still prevent some SSR errors. We should keep it.